cmd/utils/selector.go

```diff
@@ -275,8 +275,7 @@ func (s *OktetoSelector) run(initialPosition int) (string, error) {
 		return "", err
 	}
 
-	items, idx := l.Items()
-	item := items[idx]
+	item := strings.TrimSpace(strings.TrimSuffix(s.Items[l.Index()].Name, " *"))
```

Comment:
Wouldn't it be better to not save it with the `*`? We could change the `run` function to not modify the selector Items

Reply:
We need it to show what's the actual one, and we can't change the behavior of the function